manager/controllers/app/multicluster/multi_cluster_manager.go

```diff
@@ -0,0 +1,75 @@
+package multicluster
+
+import (
+	"fmt"
+	"github.com/ibm/the-mesh-for-data/manager/apis/app/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+type ClusterManager interface {
+	GetClusters() ([]Cluster, error)
+	GetBlueprint(cluster string, namespace string, name string) (*v1alpha1.Blueprint, error)
+	CreateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
+	UpdateBlueprint(cluster string, blueprint *v1alpha1.Blueprint) (DeploymentDetails, error)
+	DeleteBlueprint(cluster string, namespace string, name string) error
+}
+
+type ClusterMetadata struct {
+	Region string
+	Zone   string
```

Comment:
OK for this PR since it follows the issue. But be aware that some clusters can span across multiple zones and we will need to deal with that later.